protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -175,6 +185,11 @@ protocol SimpleFS {
 */
   void simpleFSListRecursive(OpID opID, Path path, ListFilter filter, boolean refreshSubscription);
 
+  /**
+   Begin recursive list of items in directory at path up to a given depth
+   */
+  void simpleFSListRecursiveToDepth(OpID opID, Path path, ListFilter filter, int depth);
```

Comment:
Did you mean to use `ListToDepthArgs` here rather than flattening them out?

Reply:
Nope I want them flat because it's called with either a `ListArgs` or `ListToDepthArgs`.

Reply:
Do we want `refreshSubscription` here too?